pages/linux/goldeneye.md

```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
+
+- Test a specific website without verifying ssl certficate:
```

Comment:
```suggestion
- Test a specific website without [n] verifying the SSL certificate:
```

Reply:
- Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

Reply:
```suggestion
- Test a specific website without verifying SSL certificate:
```